db/re/item_db_usable.yml

```diff
@@ -78855,6 +78902,387 @@ Body:
+    Name: ROS Korea Participant Box    # !todo check english name
+    Type: Usable
+    Flags:
+      Container: true
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getgroupitem(IG_2024ROS_BOX_4TH);
+  - Id: 103118
+    AegisName: 2024ROS_Box_5th
+    Name: ROS Korea Applicant Box    # !todo check english name
+    Type: Usable
+    Flags:
+      Container: true
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getgroupitem(IG_2024ROS_BOX_5TH);
+  - Id: 103234
+    AegisName: aegis_103234
+    Name: Costume Enchant Stone Box36    # !todo check english name
+    Type: Usable
+    Weight: 10
+    Flags:
+      Container: true
+    Script: |
+      getgroupitem(IG_AEGIS_103234);
+  - Id: 103260
+    AegisName: Summer_Select_Box1
+    Name: Summer Costume Selection Box (Pioneer)    # !todo check english name
+    Type: DelayConsume
+    Weight: 10
+  - Id: 103261
+    AegisName: Summer_Select_Box2
+    Name: Summer Costume Selection Box (Parasol)    # !todo check english name
+    Type: DelayConsume
+    Weight: 10
+  - Id: 103262
+    AegisName: Summer_Select_Box3
+    Name: Summer Costume Selection Box (Sunbed)    # !todo check english name
+    Type: DelayConsume
+    Weight: 10
+  - Id: 103263
+    AegisName: Summer_Select_Box4
+    Name: Star Footprint Selection Box (Summer Beach)    # !todo check english name
+    Type: DelayConsume
+    Weight: 10
+  - Id: 103266
+    AegisName: Kr_B_Special11
+    Name: Sweet Poring Cake    # !todo check english name
+    Type: Usable
+    Weight: 10
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+  - Id: 103272
+    AegisName: E_Speed_Booster
+    Name: "[Non-Sellable] Speed ​​Booster"    # !todo check english name
+    Type: Usable
+    Weight: 10
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+  - Id: 103273
+    AegisName: E_Force_Booster
+    Name: "[Non-Sellable] Force Booster"    # !todo check english name
+    Type: Usable
+    Weight: 10
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+  - Id: 103278
+    AegisName: 2024Sports_Box
+    Name: 2024 World Sports Cheer Box    # !todo check english name
+    Type: Usable
+    Flags:
+      Container: true
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getgroupitem(IG_2024SPORTS_BOX);
+  - Id: 103307
```

Comment:
Doesn't this item need a function in the script to activate the character slot?

Reply:
Maybe, I will leave that to you. Thanks!